### DISCUSSION TITLE
My Template is not being loaded to FranklySecure.github.io

### DISCUSSION THREAD

**Discussion by FranklySecure** [community/community#69620] [Programming Help] [answered]
### Body

Can anyone explain why this is happening? I apoligise if I posted this in the wrong place, if so can someone tell me where I would go for help with this? Sorry, very inexperienced but want to learn.
FranklySecure.github.io
[index.txt](https://github.com/community/community/files/12848685/index.txt)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebubekirgungor** [comment, marked as answer]
Hello you should change all "js/" strings to "blue-bootstrap-main/js/" and "css/" strings to "blue-bootstrap-main/css/" in your index.html file.

**Comment by ebubekirgungor** [comment]
I created a pull request for fix it.

**Reply by FranklySecure** [reply]
Thank you so much!

**Reply by ebubekirgungor** [reply]
You are welcome

**Reply by ebubekirgungor** [reply]
Can you mark the answer as answered?
